docker/build-tools/Dockerfile

```diff
@@ -79,11 +74,14 @@ FROM golang:1.12 as golang_context
 # Build a base os context
 FROM ubuntu:xenial as base_os_context
 
-RUN apt-get update
-RUN apt-get install -y make ca-certificates git
-RUN apt-get autoremove -y
-RUN apt-get clean
-RUN rm -rf /var/lib/apt/lists/*
+# hadolint ignore=DL3008
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    make \
+    ca-certificates \
+    git \
+    wget \
+    curl \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
this makes a difference (with the `rm -rf `- although chaining commands is not needed as the layers are compressed when used with a `COPY --from` operation.